javascript-data-model/order-history.js

```diff
@@ -0,0 +1,63 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    totalCost: 34,
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    deliveredDate: 'Aug 8, 2020',
+    returnWindow: 'Sep 7, 2020',
+    itemList: [{
+      name: 'JavaScript for impatient programmers',
+      type: 'book',
+      author: 'Raushmayer, Dr. Axel',
+      cost: 31.55
+    }]
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    totalCost: 44.53,
+    shipTo: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    deliveredDate: 'Jul 20, 2020',
+    returnWindow: 'Aug 19, 2020',
+    itemList: [{
+      name: 'The Timeless Way of Building',
+      type: 'book',
+      author: 'Alexander, Christopher',
+      cost: 41.33
+    }]
+  },
+  {
+    orderPlaced: 'July 4, 2020',
+    totalCost: 44.52,
+    shipTo: 'JS Masher',
+    orderNumber: '114-2875557-9059409',
+    deliveredDate: 'Jul 7, 2020',
+    returnWindow: 'Aug 5, 2020',
+    itemList: [{
+      name: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+      type: 'electronic',
+      author: null,
```

Comment:
For something like this, you can just leave this property off, since it isn't relevant to this type of item.